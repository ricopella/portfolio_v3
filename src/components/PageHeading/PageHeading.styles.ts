import styled from '@emotion/styled'

const PageHeading = styled.h1`
  color: ${props => props.theme.color};
  fontsize: 3rem;
  text-align: left;
`

export default {
  PageHeading,
}
