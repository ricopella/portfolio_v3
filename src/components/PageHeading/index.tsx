import React, { FC } from 'react'
import Styled from './PageHeading.styles'
import { PageHeadingProps } from '../../types'

const PageHeading: FC<PageHeadingProps> = ({ title }) => (
  <Styled.PageHeading>{title}</Styled.PageHeading>
)

export default PageHeading
