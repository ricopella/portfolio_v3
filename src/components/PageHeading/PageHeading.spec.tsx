import PageHeading from './index'
import React from 'react'
import renderer from 'react-test-renderer'
import { render } from '@testing-library/react'

describe('<PageHeading />', () => {
  const props = {
    title: 'Boston Bruins',
  }

  it('should match PageHeading snapshot', () => {
    const internalTree = renderer.create(<PageHeading {...props} />).toJSON()

    expect(internalTree).toMatchSnapshot()
  })

  it('should render children', () => {
    const { getByText } = render(<PageHeading {...props} />)

    expect(getByText(/Boston Bruins/i)).toBeTruthy()
  })
})
