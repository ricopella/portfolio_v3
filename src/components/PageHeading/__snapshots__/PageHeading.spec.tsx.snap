// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageHeading /> should match PageHeading snapshot 1`] = `
<h1
  className="css-onmyla"
>
  Boston Bruins
</h1>
`;
